src/components/editions/headerImageCaption.tsx

```diff
@@ -0,0 +1,116 @@
+import type { SerializedStyles } from '@emotion/core';
+import { css } from '@emotion/core';
+import { remSpace } from '@guardian/src-foundations';
+import { from } from '@guardian/src-foundations/mq';
+import { brandAlt, neutral } from '@guardian/src-foundations/palette';
+import { textSans } from '@guardian/src-foundations/typography';
+import { SvgCamera } from '@guardian/src-icons';
+import type { Option } from '@guardian/types';
+import { map, withDefault } from '@guardian/types';
+import { pipe2 } from 'lib';
+import type { FC, ReactElement } from 'react';
+
+const captionId = 'header-image-caption';
+
+const HeaderImageCaptionStyles = (
+	iconBackgroundColor?: string,
+): SerializedStyles => css`
+	summary {
+		text-align: center;
+		background-color: ${iconBackgroundColor
+			? iconBackgroundColor
+			: brandAlt[400]};
+		width: 34px;
+		height: 34px;
+		position: absolute;
+		bottom: 12px;
```

Comment:
Any reason for px here over remSpace or rem?  Should we use remspace where available and if not, fall back to rem for consistency?

Reply:
Good spot! Updated.